packages/react-conformance-make-styles/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "include": ["src"],
+  "compilerOptions": {
+    "target": "ES2020",
+    "module": "CommonJS",
+    "lib": ["ES2020", "dom"],
+    "outDir": "dist",
+    "jsx": "react",
+    "declaration": true,
+    "experimentalDecorators": true,
+    "importHelpers": true,
+    "noUnusedLocals": true,
+    "preserveConstEnums": true,
+    "types": ["jest", "custom-global", "inline-style-expand-shorthand", "storybook__addons", "node"]
```

Comment:
I'm not sure all of these are needed here? Unless we're trying to standardize the list even when not all of them are used.

Reply:
I removed `storybook__addons` from there as it's unused for sure (will be fixed by #19459). This is produced by `nx generator`, if you there something redundant I guess there is sense to create an issue for this.